### PR TITLE
Add versionRemoved tag to trees.config

### PR DESCRIPTION
**Pull request by poornimanayar**
The trees.config file is not applicable to v8. So I have added the tag versionRemoved: 8.0.0

### CODE REVIEW

**Comment by dampee** [comment]
Hi @poornimanayar 

True, but we should help docs users also on where to start. when they want to create a new tree.  By not adding an alternative we suggest it is not possible anymore.  And we still need to "update" the document for v8.

Any chance you could create a small 2 paragraph start for this?

**Comment by dampee** [comment]
Oh, and same for #2164  and #2165

**Comment by poornimanayar** [comment]
Sure can @dampee.

**Comment by sofietoft** [comment]
@poornimanayar - Check out my comment here: #2163 (comment) 😁

File: Reference/Config/trees/index.md [hunk 1]
```diff
@@ -1,68 +1,6 @@
...
-  <add initialize="true" sortOrder="1" alias="memberTypes" application="member" title="Member Types" iconClosed="icon-folder" iconOpen="icon-folder-open" type="Umbraco.Web.Trees.MemberTypeTreeController, umbraco" />
-  <add application="member" sortOrder="2" alias="memberGroups" title="Member Groups" type="umbraco.loadMemberGroups, umbraco" iconClosed="icon-folder" iconOpen="icon-folder" />
-  <!--Translation-->
-  <add silent="false" initialize="true" sortOrder="1" alias="openTasks" application="translation" title="Tasks assigned to you" iconClosed="icon-folder" iconOpen="icon-folder" type="umbraco.loadOpenTasks, umbraco" />
-  <add silent="false" initialize="true" sortOrder="2" alias="yourTasks" application="translation" title="Tasks created by you" iconClosed="icon-folder" iconOpen="icon-folder" type="umbraco.loadYourTasks, umbraco" />
-  <!-- Custom entries appear here -->
-</trees>
-```
+You can implement a custom tree using code - [Implementing Custom Trees]((../../../Extending/Section-Trees/trees))
```
Comment (sofietoft) [inline Reference/Config/trees/index.md]: Added some more text to make the link a bit more accessible - what do you think?

```suggestion
You can implement a custom tree using code. Learn more about this approach in the [article about Section Trees](../../../Extending/Section-Trees/trees).
```

File: Reference/Config/trees/index-v7.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+versionFrom: 7.0.0
```
Comment (sofietoft) [inline Reference/Config/trees/index-v7.md]:
```suggestion
versionFrom: 7.0.0
versionRemoved: 8.0.0
```

**Comment by poornimanayar** [comment]
Updated @sofietoft  :-)

**Comment by sofietoft** [comment]
Great, thanks! 😄 

Let's get this merged then 💪

**Pull request merged**